build.gradle

```diff
@@ -0,0 +1,39 @@
+ext {
+  kotlinVersion = '1.0.4'
+
+  compileSdkVersion = 24
+  buildToolsVersion = '24.0.2'
+
+  dep = [
+      androidPlugin: 'com.android.tools.build:gradle:2.2.0',
+      supportAnnotations: 'com.android.support:support-annotations:24.2.1',
+      javaPoet: 'com.squareup:javapoet:1.7.0',
+      kotlinStdLib: "org.jetbrains.kotlin:kotlin-stdlib:$kotlinVersion",
+      kotlinReflect: "org.jetbrains.kotlin:kotlin-reflect:$kotlinVersion",
+      kotlinGradlePlugin: "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlinVersion",
+      junit: 'junit:junit:4.12',
+      retrolambda: 'me.tatarka:gradle-retrolambda:3.3.0'
```

Comment:
Curious what we see retroLambda being used for? Just want to make sure its worth it since we're including kotlin as well with nice lambda support.